src/router/routes.ts

```diff
@@ -31,12 +33,12 @@ const routes: Array<RouteConfig> = [
     }
   },
   {
-    path: '/about',
-    name: 'About',
+    path: '/dashboard',
+    name: 'Dashboard',
     // route level code-splitting
-    // this generates a separate chunk (about.[hash].js) for this route
+    // this generates a separate chunk (dashboard.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')
+    component: () => import(/* webpackChunkName: "dashboard" */ '../views/Dashboard.vue')
```

Comment:
may be we don't need lazy loading ;staff application rite

Reply:
this will make different chunks after build. Hope it will be good